Clic&Farm-iOSMockTests/ApolloTests.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ApolloTests.swift
+//  Clic&Farm-iOSMockTests
+//
+//  Created by Guillaume Roux on 16/11/2018.
+//  Copyright © 2018 Ekylibre. All rights reserved.
+//
+
+import XCTest
+@testable import Clic_Farm_iOS
+import Apollo
+
+class ApolloTests: XCTestCase {
+
+  override func setUp() {
```

Comment:
Overriding to call super seems unnecessary. Is this really useful?

Reply:
We didn't make Apollo tests yet, that's why this class is useless. Removed mock tests from project.